README.md

```diff
@@ -4,41 +4,170 @@ Please edit this README.md file with your answers to the worksheet questions.
+|   0   |    1    |    0    |        0        |         0        |                    0                   |
+|   0   |    1    |    1    |        0        |         1        |                    1                   |
+|   1   |    0    |    0    |        0        |         0        |                    0                   |
+|   1   |    0    |    1    |        1        |         0        |                    1                   |
+|   1   |    1    |    0    |        0        |         0        |                    0                   |
+|   1   |    1    |    1    |        1        |         1        |                    1                   |
+
+|  Int or Float  |  (Int or Float) and x > 7  |
+|----------------|----------------------------|
+|        0       |             0              |
+|        0       |             0              |
+|        1       |             0              |
+|        1       |             1              |
+|        1       |             0              |
+|        1       |             1              |
+|        1       |             0              |
+|        1       |             1              |
 ### c
-
+|  x = 0  |  y = 0  |  x = 0 and y = 0  |
+|---------|---------|-------------------|
+|    0    |    0    |         0         |
+|    0    |    1    |         0         |
+|    1    |    0    |         0         |
+|    1    |    1    |         1         |
+
+The second bit of code does nothing unless both x and y = 0
 ### d
+|  x > 10  |  x > 0  |  y > 0  |  x > 0 and y > 0  |  x > 10 or (x > 0 and y > 0)  |
+|----------|---------|---------|-------------------|-------------------------------|
+|     0    |    0    |    0    |         0         |               0               |
+|     0    |    0    |    1    |         0         |               0               |
+|     0    |    1    |    0    |         0         |               0               |
+|     0    |    1    |    1    |         1         |               1               |
+|     1    |    0    |    0    |         0         |               1               |
+|     1    |    0    |    1    |         0         |               1               |
+|     1    |    1    |    0    |         0         |               1               |
+|     1    |    1    |    1    |         1         |               1               |
+
+
+|  x > 10 or y > 0  |  x > 0 and (x > 10 or y > 0)  |
+|-------------------|-------------------------------|
+|         0         |               0               |
+|         1         |               0               |
+|         0         |               0               |
+|         1         |               1               |
+|         1         |               0               |
+|         1         |               0               |
+|         1         |               1               |
+|         1         |               1               |
+
+The second code was would not equal the same results as the first code.
```

Comment:
In practice it would -- the cases where the truth tables differ are those where `x>10` and not `x>0` -- which is impossible in practice, so these cases can be ignored